src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -899,13 +899,31 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
                                                 thisInfiltration.Name,
                                                 cAlphaFieldNames(4),
                                                 cNumericFieldNames(1)));
+                    } else {
+                        Real64 spaceFrac = 1.0;
+                        if (!thisInfiltrationInput.spaceListActive && (thisInfiltrationInput.numOfSpaces > 1)) {
+                            Real64 const zoneVolume = state.dataHeatBal->Zone(zoneNum).Volume;
+                            if (zoneVolume > 0.0) {
+                                spaceFrac = thisSpace.Volume / zoneVolume;
+                            } else {
+                                ShowSevereError(state, std::string(RoutineName) + "Zone volume is zero when allocating Infiltration to Spaces.");
```

Comment:
I understand that `string + literal string` is a special case, but this is still at best one reallocation more than `format`.